### ISSUE TITLE
GDC: add reporting details

### ISSUE THREAD

**Issue by tomkralidis** [wmo-im/wis2-guide#170]
Add technical considerations for GDC reporting on metadata publication (conformance testing, KPIs, etc.).

**Comment by tomkralidis** [comment]
Related implementation has been added to pywiscat (https://github.com/wmo-im/pywiscat/pull/13), with the approach of a standalone tool that can analyze any WIS2 GDC per the [Metrics Hierarchy](https://github.com/wmo-im/wis2-metric-hierarchy/blob/main/metrics/gdc.csv) via CLI or Python library (dumping JSON).  This will be made available once the work is approved/merge and a new pywiscat release is made (in the next days).

The [wis2-gdc](https://github.com/wmo-im/wis2-gdc) RI will then leverage this functionality to generate OpenMetrics accordingly.

**Comment by 6a6d74** [comment]
@tomkralidis - is this action complete (i.e., updates to the Guide are merged into INFCOM-4 branch)? Can it be closed?
